hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/storage/HoodieFileWriterFactory.java

```diff
@@ -71,7 +71,7 @@
 
     HoodieAvroParquetConfig parquetConfig = new HoodieAvroParquetConfig(writeSupport, config.getParquetCompressionCodec(),
         config.getParquetBlockSize(), config.getParquetPageSize(), config.getParquetMaxFileSize(),
-        hoodieTable.getHadoopConf(), config.getParquetCompressionRatio());
+        hoodieTable.getHadoopConf(), config.getParquetCompressionRatio(), Boolean.parseBoolean(config.getProps().getProperty("hoodie.parquet.dictionary.enabled", "true")));
```

Comment:
we can move `Boolean.parseBoolean(config.getProps().getProperty("hoodie.parquet.dictionary.enabled", "true"))` into HoodieWriteConfig and just call `config.parquetDictionaryEnabled` here

Reply:
+ 1

Reply:
done